extensions/renderer/resources/guest_view/web_view/web_view.js

```diff
@@ -148,11 +148,12 @@ WebViewImpl.prototype.onFrameNameChanged = function(name) {
 // Updates state upon loadcommit.
 WebViewImpl.prototype.onLoadCommit = function(
     baseUrlForDataUrl, currentEntryIndex, entryCount,
-    processId, url, isTopLevel) {
+    processId, url, isTopLevel, pagesHistory) {
   this.baseUrlForDataUrl = baseUrlForDataUrl;
```

Comment:
This would send the whole history from browser to renderer process on every navigation, which could hurt performance. Is there any better way to do this?